cypress/e2e/10-support-manage-facility.cy.js

```diff
@@ -55,33 +66,31 @@ describe("Support admin: manage facility", () => {
 
     // selects facility combo box
     cy.wait("@GetFacilitiesByOrgId");
-    cy.get('input[role="combobox"]')
-      .last()
-      .type(`${facilityCreated.name}{enter}`);
+    cy.get('input[role="combobox"]').last().type(`${facilityName}{enter}`);
 
     // clicks search button
     cy.get("button").contains("Search").click();
 
     // displays facility information
     cy.wait("@GetFacilityStats");
-    cy.contains(facilityCreated.name);
+    cy.contains(facilityName);
     cy.contains("Facility information");
     cy.contains("Facility controls");
     cy.checkAccessibility();
 
     // checks the confirmation modal
     cy.get("button").contains("Delete facility").click();
-    cy.contains(`Delete ${facilityCreated.name}`);
+    cy.contains(`Delete ${facilityName}`);
     cy.checkAccessibility();
     cy.get("button").contains("No, go back").click();
-    cy.contains(`Delete ${facilityCreated.name}`).should("not.exist");
+    cy.contains(`Delete ${facilityName}`).should("not.exist");
   });
 
   it("Deletes a facility", () => {
```

Comment:
nit: I know this wasn't introduced here but I'm wondering if the loading and deleting a facility should be within one `it` block since it makes sense to me that is the entire flow we are testing 🤔